framework/decode/vulkan_replay_consumer_base.cpp

```diff
@@ -9703,9 +9703,7 @@ VulkanAddressReplacer& VulkanReplayConsumerBase::GetDeviceAddressReplacer(const
 
 bool VulkanReplayConsumerBase::UseExtraDescriptorInfo(const VulkanDeviceInfo* device_info) const
 {
-    // TODO: testing with an override here
-    return true;
-//    return options_.dumping_resources || !device_info->allocator->SupportsOpaqueDeviceAddresses();
+    return options_.dumping_resources || !device_info->allocator->SupportsOpaqueDeviceAddresses();
```

Comment:
This is specific to the general "opaque address support"?  Or does it look at all of the feature flags?  My concern is that it might be present for one case, but not all the cases you use this check to cover.